services/postgresql/CHANGELOG.md

```diff
@@ -2,9 +2,11 @@
 >
 > Use `github.com/stackitcloud/stackit-sdk-go/services/postgresflex` instead. For more details, check https://docs.stackit.cloud/stackit/en/bring-your-data-to-stackit-postgresql-flex-138347648.html.
 
-## v0.13.1 (2024-04-09)
+## v0.14.0 (2024-04-11)
```

Comment:
Remove this entry from the changelog. Since PG is deprecated, no further releases should be done